dts/bindings/led_strip/worldsemi,ws2812-rpi_pico-pio.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023, TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Worldsemi WS2812 LED strip, RaspberryPi Pico's PIO binding
+
+  Driver bindings for controlling a WS2812 or compatible LED
+  strip with a PIO feature.
+
+  Parent PIO device's configure the pin that use for output signals.
+
+compatible: "worldsemi,ws2812-rpi_pico-pio"
+
+include: [base.yaml, ws2812.yaml, "raspberrypi,pico-pio-device.yaml"]
+
+properties:
+  frequency:
+    type: int
+    default: 800000
```

Comment:
Does the driver support a different frequency ? I don't see in the driver how the signal is adjusted according to the frequency.

Reply:
Yes but then changing the frequency also changes the duration of the pulses output by the PIO program. AFAIU (and I don't understand much :) the pulses seems to be "hardcoded" by the PIO program instructions. So I suspect that using a different frequency won't work.

Reply:
I see -- if that's the case, there seems to be no need for this property.

Reply:
This property intends  to supporting WS2811(400kHz), WS2813 or WS2815(2MHz).
It is inherit from https://github.com/raspberrypi/pico-examples/tree/master/pio/ws2812, not yet tested excluding 800000 in zephyr.

Reply:
So how does this frequency affect the pulses ?

Let's say a user selects the 400kHz frequency for a strip of WS2811 devices. In that case is it still correct for the PIO program to output `0001100000` and `0001111111` pulses for 0 and 1 bits ?

Did you consider to make the pulses configurable as in the SPI driver (see the `spi-one-frame` and `spi-zero-frame `DT properties) ? This would make this driver compatible with more LED controllers.

Reply:
I understood it that is the optimal value is different by LED models.
But I'm not have other than ws2812.
I'm ordering ws2813 now. I'll consider to add the option for tune to particular LED model.

Reply:
So how does this frequency affect the pulses ?

Let's say a user selects the 400kHz frequency for a strip of WS2811 devices. In that case is it still correct for the PIO program to output `0001100000` and `0001111111` pulses for 0 and 1 bits ?

Did you consider to make the pulses configurable as in the SPI driver (see the `spi-one-frame` and `spi-zero-frame` DT properties) ? This would make this driver compatible with more LED controllers.

Reply:
> I understood it that is the optimal value is different by LED models. But I'm not have other than ws2812. I'm ordering ws2813 now. I'll consider to add the option for tune to particular LED model.

The frequency "probably" affects the width pulse of a bit. So I don't understand how hardcoded pulse values for 0 and 1 bits can work with different frequencies. Indeed it would be interesting to test with another model.